src/redux/actions/messages.js

```diff
@@ -0,0 +1,15 @@
+import makeEnum from '../lib/make_enum';
+
+export const types = makeEnum([
+  'SYNC_MESSAGES',
```

Comment:
@tomclose happy to follow the nested convention you were using elsewhere, if you prefer that (like `types.MESSAGES.SYNC`. Seemed unnecessary to have it nested for messages so did it this way.